ocs_ci/deployment/deployment.py

```diff
@@ -2654,6 +2659,8 @@ def deploy(self):
         self.configure_mirror_peer()
         # Deploy dr policy
         self.deploy_dr_policy()
+        # Configure DRClusters for fencing automation
+        configure_drcluster_for_fencing()
```

Comment:
This should be done only for MDR

Reply:
This is done inside MDR class [2644](https://github.com/red-hat-storage/ocs-ci/pull/7978/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fL2639)

Reply:
Okay